### DISCUSSION TITLE
Feature Request: CSV export for String Tables

### DISCUSSION THREAD

**Discussion by korgano** [4sval/FModel#494] [Q&A]
Currently, the only export options for String Tables in Fmodel is a dump to JSON or a raw uAsset dump.

However, UE4/5 support importing CSV files into String Tables, which speeds up a number of tasks:

- Rebuilding files for mods with string table references.
- Updating the contents of string tables to correct errors and add new content.
- Creating localizations.

 Adding CSV export for game String Tables would allow mod devs to reap the benefits of this UE functionality, instead of spending time converting the JSON to CSV.

**Comment by GMatrixGames** [comment]
![image](https://github.com/user-attachments/assets/9432687d-47a6-471d-8ab7-950044952201)
